enterprise/frontend/src/metabase-enterprise/whitelabel/lib/whitelabel.js

```diff
@@ -5,6 +5,7 @@ import Color from "color";
 import { colors } from "metabase/lib/colors/palette";
 import { addCSSRule } from "metabase/lib/dom";
 
+import { omit } from "lodash";
```

Comment:
What's your view on using `lodash` versus `underscore`. I've had trouble when trying to use some utilities from these libraries, especially, when desired functions exist in both libraries.

I'm not sure if there's a consensus on which one to pick from first.

Reply:
Based on latest conversations in the FE guild, we decided to go for `lodash` because it's better maintained.